SI/Sem.02/Pract.02/task_8_p02.cpp

```diff
@@ -0,0 +1,33 @@
+
+
+#include <iostream>
+
+int main()
+{
+    int number = 0;
```

Comment:
You don't need to intialize the variable with value, because you will overwrite it with the cin